src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball firstBall = new Lottery().getRandomBall();
+        Ball secondBall = new Lottery().getRandomBall();
+        Ball thirdBall = new Lottery().getRandomBall();
+        System.out.println("First ball. Color - " + firstBall.getColor() + ", number - "
+                + firstBall.getNumber() + ".");
+        System.out.println("Second ball. Color - " + secondBall.getColor() + ", number - "
+                + secondBall.getNumber() + ".");
+        System.out.println("Third ball. Color - " + thirdBall.getColor() + ", number - "
+                + thirdBall.getNumber() + ".");
```

Comment:
let's override `toString()` method in `Ball` class where we will return information about balls (remove `first`, `second` words)